addons/project/views/project_task_views.xml

```diff
@@ -147,6 +147,7 @@
                 'default_project_id': active_id,
                 'show_project_update': True,
                 'search_default_open_tasks': 1,
+                'task_view': True,
```

Comment:
If we do end up going this route however, we should add the context key in other places, because as is, it won't be active if you go through "My Tasks" or "All Tasks", or even when accessing the task directly through the link